Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.h

```diff
@@ -33,6 +33,7 @@ namespace Terrain
         AZ_CLASS_ALLOCATOR(TerrainPhysicsSurfaceMaterialMapping, AZ::SystemAllocator, 0);
         AZ_RTTI(TerrainPhysicsSurfaceMaterialMapping, "{A88B5289-DFCD-4564-8395-E2177DFE5B18}");
         static void Reflect(AZ::ReflectContext* context);
+        static AZ::Data::AssetId GetMaterialLibraryId();
```

Comment:
Please make this function private, because it's only meant to be used internally for the reflection function.

Reply:
tbh since it's a struct rather than a class, it's more assumed to be all-public data rather than an object with encapsulation rules. But added a private section for the function if you feel that's better

Reply:
Yes, that's what I meant, make the function only private. Thanks.